<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateArticle</class>
 <widget class="QWidget" name="CreateArticle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="articetitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TitlelineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DescriptionlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DatelineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CategorylineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RatinglineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateArticleButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Article</string>
   </property>
  </widget>
  <widget class="QLabel" name="CreateArticleUpdate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CreateArticleupdate">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
